application/ticket-app-api/src/main/java/com/uket/app/ticket/api/controller/TicketApi.java

```diff
@@ -88,4 +93,18 @@ ResponseEntity<TicketingResponse> ticketing(
             @RequestBody
             TicketingRequest request
     );
+
+    @Operation(summary = "사용자 티켓 조회 API", description = "특정 사용자의 모든 티켓을 조회합니다.")
+    @GetMapping("/all/{userId}")
+    @ApiResponse(responseCode = "200", description = "OK", content = @Content(array = @ArraySchema(schema = @Schema(implementation = CheckTicketDto.class))))
+    @ApiResponse(responseCode = "404", description = "NOT FOUND", content = @Content(mediaType = "application/json",
+        examples = @ExampleObject(name = "US0001", description = "토큰에 담긴 UserId에 대한 사용자를 찾을 수 없을 때 발생합니다.",
+            value = """
+                        {"code": "US0001", "message": "해당 사용자를 찾을 수 없습니다."}
+                        """
+        ), schema = @Schema(implementation = ErrorResponse.class)))
+    List<CheckTicketDto> getUserTickets(
+        @Parameter(description = "사용자 ID")
+        @PathVariable Long userId
+    );
```

Comment:
저희 응답 주는 방식은 ResponseEntity를 사용하는 걸로 통일할게요! List로 응답이 필요한 경우에는 `dto/response` 패키지 안에 ListResponse있는데 그거 사용하면 좋을 것 같아용
```suggestion
    ResponseEntity<ListResponse<CheckTicketDto>> getUserTickets(
        @Parameter(description = "사용자 ID")
        @PathVariable Long userId
    );
```